examples/calling-apis/spa-with-backend-api/react-hono-ai-sdk/package.json

```diff
@@ -11,9 +12,7 @@
   "workspaces": [
     "./server",
     "./client",
-    "./shared",
-    "../../../../packages/ai",
-    "../../../../packages/ai-vercel"
+    "./shared"
```

Comment:
dependent package builds no longer needed here, we simply rely on workspace symlinks now